Server/src/genericInterfaces/ResponsesInterface.ts

```diff
@@ -0,0 +1,19 @@
+export interface IResponse {
+    status: string
+    statusCode: number
+    response?: string
+}
+
```

Comment:
the file could be named ResponsesInterfaces.ts to follow AuthenticationInterfaces.ts or just ResponseInterface, however you choose

Reply:
The interfaces and exceptions will still be worked on